release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1247,6 +1247,19 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* If the installation program is unable to get all of the projects that are associated with the Google Cloud Platform (GCP) service account, the installation fails with the following error message:
++
+[source,text]
+----
+FATAL failed to fetch Install Config: failed to fetch dependency of "Install Config": failed to fetch dependency of "Base Domain": failed to generate asset "Platform": failed to get projects: context deadline exceeded
+----
++
+This behavior occurs when the following conditions are met:
+** The service account has access to an excessive number of projects.
+** The installation program is run with `create cluster` and `create manifests`, without specifying an existing installation configuration file (`install-config.yaml`).
++
+As a workaround, manually create an `install-config.yaml` file, and enter a specific project id to use (`platform.gcp.projectID`). Run the installation program again, referencing the existing file.
```

Comment:
Technically speaking, we don't reference the `install-config.yaml` file directly, it just needs to be in the directory used as workplace.